policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/dependent_care_benefit/hi_min_head_spouse_earned.py

```diff
@@ -0,0 +1,71 @@
+from policyengine_us.model_api import *
+
+
+class hi_min_head_spouse_earned(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii minimum income between head and spouse for the CDCC"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=28"
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=29"
+        "https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=41"
+        "https://files.hawaii.gov/tax/forms/2022/schx_i.pdf#page=2"
+    )
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.credits.cdcc
+        person = tax_unit.members
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        head_or_spouse = head | spouse
+        # Head or spouse are eligible for an income floor if disabled or a student
+        eligible = head_or_spouse & (
```

Comment:
```suggestion
        income_floor_eligible = head_or_spouse & (
```
Otherwise suggests eligibility for the full program